src/ImageSharp/Formats/Tiff/TiffFrameMetadata.cs

```diff
@@ -0,0 +1,331 @@
+        public uint[] StripOffsets => this.GetArray<uint>(ExifTag.StripOffsets);
+
+        /// <summary>
+        /// Gets the number of components per pixel.
+        /// </summary>
+        public ushort SamplesPerPixel => this.GetSingle<ushort>(ExifTag.SamplesPerPixel);
+
+        /// <summary>
+        /// Gets the number of rows per strip.
+        /// </summary>
+        public uint RowsPerStrip => this.GetSingle<uint>(ExifTag.RowsPerStrip);
+
+        /// <summary>
+        /// Gets for each strip, the number of bytes in the strip after compression.
+        /// </summary>
+        public uint[] StripByteCounts => this.GetArray<uint>(ExifTag.StripByteCounts);
+
+        /// <summary>Gets the resolution of the image in x- direction.</summary>
+        /// <value>The density of the image in x- direction.</value>
+        public double? HorizontalResolution
+        {
+            get
+            {
+                if (this.ResolutionUnit != TiffResolutionUnit.None)
+                {
+                    double resolutionUnitFactor = this.ResolutionUnit == TiffResolutionUnit.Centimeter ? 2.54 : 1.0;
+
+                    if (this.TryGetSingle(ExifTag.XResolution, out Rational xResolution))
+                    {
+                        return xResolution.ToDouble() * resolutionUnitFactor;
+                    }
+                }
+
+                return null;
+            }
+        }
+
+        /// <summary>
+        /// Gets the resolution of the image in y- direction.
+        /// </summary>
+        /// <value>The density of the image in y- direction.</value>
+        public double? VerticalResolution
+        {
+            get
+            {
+                if (this.ResolutionUnit != TiffResolutionUnit.None)
+                {
+                    double resolutionUnitFactor = this.ResolutionUnit == TiffResolutionUnit.Centimeter ? 2.54 : 1.0;
+
+                    if (this.TryGetSingle(ExifTag.YResolution, out Rational yResolution))
+                    {
+                        return yResolution.ToDouble() * resolutionUnitFactor;
+                    }
+                }
+
+                return null;
+            }
+        }
+
+        /// <summary>
+        /// Gets how the components of each pixel are stored.
+        /// </summary>
+        public TiffPlanarConfiguration PlanarConfiguration => this.GetSingleEnum<TiffPlanarConfiguration, ushort>(ExifTag.PlanarConfiguration, DefaultPlanarConfiguration);
+
+        /// <summary>
+        /// Gets the unit of measurement for XResolution and YResolution.
+        /// </summary>
+        public TiffResolutionUnit ResolutionUnit => this.GetSingleEnum<TiffResolutionUnit, ushort>(ExifTag.ResolutionUnit, DefaultResolutionUnit);
+
+        /// <summary>
+        /// Gets the name and version number of the software package(s) used to create the image.
+        /// </summary>
+        public string Software => this.GetString(ExifTag.Software);
```

Comment:
How are all these values supposed to be updated?

Reply:
@JimBobSquarePants yes, the values supposed to be updated, but do not support saving to tiff yet, I plan to add support for updating when there will be support for saving

Reply:
Cool. Let's see what we come up with to hold frame EXIF metadata